### ISSUE TITLE
Embedded samples: pre-selected tab doesn't have aria-selected attribute

### ISSUE THREAD

**Issue by bpaduch** [ampproject/amp-by-example#880]
When an embedded sample displays, *before actually clicking a tab*, the pre-selected tab (most cases, it's "Source") isn't selected, and doesn't have `aria-selected="true"`.  However, the section for the tab is pre-selected and does have the attribute (that is, "aria-selected=true" is present on the section). The  "aria-selected=true" attribute is needed on the anchor for the ability to style tabs per https://github.com/ampproject/docs/issues/510.

To replicate:

1.  Go to https://www.ampproject.org/docs/guides/responsive/responsive_design#example:-embedding-a-youtube-video.
2. Inspect "Source" tab.  Anchor has `<a href="#source-panel" role="tab" aria-control="source-panel">Source</a>`.
3. Click/select "Source" tab and inspect anchor: `<a href="#source-panel" role="tab" aria-control="source-panel" tabindex="0" aria-selected="true">Source</a>`.
